code/game/area/areas/shuttles.dm

```diff
@@ -139,6 +139,7 @@
 /area/shuttle/supply
 	name = "Supply Shuttle"
 	blob_allowed = FALSE
+	noteleport = TRUE
```

Comment:
If there's a way for the noteleport variable to check for being on the Centcom/restricted z-level that'd be more convenient, otherwise may as well do the same to fix #31429.